src/dmd/opover.d

```diff
@@ -620,9 +620,14 @@ Expression op_overload(Expression e, Scope* sc, TOK* pop = null)
                     // Old way, kept for compatibility with D1
                     if (e.op != TOK.prePlusPlus && e.op != TOK.preMinusMinus)
                     {
-                        fd = search_function(ad, opId(e));
+                        auto id = opId(e);
+                        fd = search_function(ad, id);
                         if (fd)
                         {
+                            // @@@DEPRECATED_2.094@@@.
+                            // Deprecated in 2.088
+                            // Make an error in 2.094
+                            e.deprecation("`%s` is deprecated.  Use `opUnary` instead.", id.toChars());
```

Comment:
Likewise, can we have a message which shows `opUnary!("%s")` ? This is less of a concern because we have the operator name, but things like `Shr`(or even `opStar` TBH) are not commonly understood.